systemTimers.h

```diff
@@ -4,5 +4,6 @@
 #include <Arduino.h>
 
 const unsigned int FSM_TIMER = RAK_TIMER_0; // Timer to tell how long system to sleep for during each state
+const unsigned int UPLINK_PROCESS_TIMER = RAK_TIMER_1; // Timer to change the uplink process to false after a certain amount of time
```

Comment:
I was going to ask you whether this could be done instead of using `delay`, but now I see that you've commented this out elsewhere. Did it not work like you expected?

Reply:
I have tried using a timer to avoid using `delay`, but this significantly reduces uplink success consistency.